pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -489,15 +489,23 @@ func (k *Kubernetes) ConfigEnvs(name string, service kobject.ServiceConfig) []ap
 // CreateKubernetesObjects generates a Kubernetes artifact for each input type service
 func (k *Kubernetes) CreateKubernetesObjects(name string, service kobject.ServiceConfig, opt kobject.ConvertOptions) []runtime.Object {
 	var objects []runtime.Object
+	var replica int
+	if opt.Replicas > 1 {
```

Comment:
like what I mentioned before, would be better to do:

```
if opt.IsReplicaSetFlag {
  replica = opt.Replicas
}
```

So that means that if it has been changed from the default value, it'd pick it up, even if the user puts `replicas=1`.

You will have to add another flag similar to the other ones here: https://github.com/kubernetes-incubator/kompose/blob/master/cmd/convert.go#L84